docs/kotlin.md

```diff
@@ -35,6 +35,7 @@ Add the Kotlin classpath to `buildscript.dependencies`
 ```
 dependencies {
     ...
+    classpath("com.android.tools.build:gradle:7.0.2") // or recent
```

Comment:
```suggestion
    classpath('com.android.tools.build:gradle:4.1.3') // or recent
```
 is known to work. Would you wanna change this to 4.1.3? I'm not sure about more recent versions of gradle - they come with a lot of other changes required.